src/pages/transaction/index.tsx

```diff
@@ -143,19 +145,24 @@ export const TransactionPage: React.FC = () => {
         <Flex>
           <Box flexGrow={1}>
             <Text textStyle="caption" color="ink.600">
-              Fees
+              {signedTransaction.value && sponsored ? 'No fees' : 'Fees'}
```

Comment:
I know you're going to change this to match the figma, but this can be left to the original of always just "fees"